packages/suite-desktop/src-electron/app.ts

```diff
@@ -161,6 +161,25 @@ const init = async () => {
     // repeated during app lifecycle (e.g. Ctrl+R)
     ipcMain.handle('handshake/load-modules', (_, payload) => loadModulesResponse(payload));
 
+    const loadTor = await initTorModule({
```

Comment:
I would maybe like to see a comment here why is tor loading done separately here instead of in general `initModules` function. so that future generations know

Reply:
Yes, our future us will be happy to know why this is like that afceb11d93725f575d71fb0762becde7c6e7bdf8